storage/store.go

```diff
@@ -500,11 +520,24 @@ func (s *Store) maybeSplitRangesByConfigs(configMap PrefixConfigMap) {
 	}
 }
 
+// ForceReplicationScan iterates over all ranges and enqueues any that need to be
+// replicated.
+func (s *Store) ForceReplicationScan() {
+	// TODO(bdarnell): scanning all ranges with the lock held could cause perf
```

Comment:
I don't expect this will be used from code in production, so I feel like you can remove this TODO.